### PR TITLE
📖  docs: update the release-1.5.md with release team info

### PR DESCRIPTION
**Pull request by joekr**
**What this PR does / why we need it**:
This adds the new release team info for the 1.5 release docs

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Review by ykakarap — COMMENTED** [review]
The formatting seems off.

**Comment by joekr** [comment]
> The formatting seems off.

great catch. Should be updated now.

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]
CI Signal/Bug Triage/Automation Manager shadows seem to have a duplicate name

File: docs/release/releases/release-1.5.md [hunk 1]
```diff
@@ -26,4 +26,8 @@ After the `.0` release monthly patch release will be created.
...
+| Communications/Docs/Release Notes Manager | Nawaz Hussain Khazielakha ([@nawazkh](https://github.com/nawazkh) ✉️ `@nawazkh`)                                | Guillermo Gaston ([@g-gaston](https://github.com/g-gaston) ✉️ ️`@ggaston`) <br> Johannes Frey ([@johannesfrey](https://github.com/johannesfrey) ✉️ ️`@jotef`) <br> Amit Kumar ([@hackeramitkumar](https://github.com/hackeramitkumar) ✉️ `@Amit Kumar`)                             |
+| CI Signal/Bug Triage/Automation Manager   | killian Muldoon ([@killianmuldoon](https://github.com/killianmuldoon) ✉️ `@Killian`) | Tobias Giese ([@tobiasgiese](https://github.com/tobiasgiese) ✉️ `@tobiasgiese`) <br> muhammad adil ghaffar ([@adilGhaffarDev](https://github.com/adilGhaffarDev) ✉️ `@adil`) <br> Christian Schlotter ([@chrischdi](https://github.com/chrischdi) ✉️ `@chrischdi`) <br> muhammad adil ghaffar ([@adilGhaffarDev](https://github.com/adilGhaffarDev) ✉️ `@adil`) <br> Aniruddha Basak ([@aniruddha2000](https://github.com/aniruddha2000) ✉️ `@aniruddha`) |
```
Comment (killianmuldoon) [inline docs/release/releases/release-1.5.md:33]:
```suggestion
| Release Lead                              | Joe Kratzat ([@joekr](https://github.com/joekr) ✉️ `@Joe Kratzat`) | Furkat Gofurov ([@furkatgofurov7](https://github.com/furkatgofurov7) ✉️ ️`@Furkat Gofurov`) <br> Prajyot Parab ([@Prajyot-Parab](https://github.com/Prajyot-Parab) ✉️ ️`@Prajyot Parab`) <br> Deepak Sharma ([@sadysnaat](https://github.com/sadysnaat) ✉️ `@Deepak Sharma`)                                               |
| Communications/Docs/Release Notes Manager | Nawaz Hussain Khazielakha ([@nawazkh](https://github.com/nawazkh) ✉️ `@nawazkh`)                                | Guillermo Gaston ([@g-gaston](https://github.com/g-gaston) ✉️ ️`@ggaston`) <br> Johannes Frey ([@johannesfrey](https://github.com/johannesfrey) ✉️ ️`@jotef`) <br> Amit Kumar ([@hackeramitkumar](https://github.com/hackeramitkumar) ✉️ `@Amit Kumar`)                             |
| CI Signal/Bug Triage/Automation Manager   | Killian Muldoon ([@killianmuldoon](https://github.com/killianmuldoon) ✉️ `@Killian`) | Tobias Giese ([@tobiasgiese](https://github.com/tobiasgiese) ✉️ `@tobiasgiese`) <br> muhammad adil ghaffar ([@adilGhaffarDev](https://github.com/adilGhaffarDev) ✉️ `@adil`) <br> Christian Schlotter ([@chrischdi](https://github.com/chrischdi) ✉️ `@chrischdi`) <br> Aniruddha Basak ([@aniruddha2000](https://github.com/aniruddha2000) ✉️ `@aniruddha`) |
```

**Review by killianmuldoon — COMMENTED** [review]
/lgtm

**Comment by fabriziopandini** [comment]
/lgtm
/approve
🥳

**Pull request merged**